### ISSUE TITLE
unhandled exception - TypeError

### ISSUE THREAD

**Issue by skenaja** [dchaplinsky/german_registry_parser#8]
error message: `TypeError: 'in <string>' requires string as left operand, not NoneType`

notice causing error: https://www.handelsregisterbekanntmachungen.de/en/skripte/hrb.php?rb_id=252215&land_abk=by

```
{"notice_id":"252215","federal_state":"by","amtsgericht":"Memmingen","aktenzeichen":"HRB 11041","posted_on":"2008-06-26","full_text":"Amtsgericht Memmingen Aktenzeichen: HRB 11041 \nBekannt gemacht am: 26.06.2008 12:00 Uhr\n\n\n\n\nDie in ( ) gesetzten Angaben zur Geschäftsanschrift und des Geschäftszweiges erfolgen ohne Gewähr.\n\n25.06.2008\n\nFTS-Frästechnik \u0026 Service GmbH, Vöhringen (Heerstr. 30, 89269 Vöhringen). Die Gesellschafterversammlung vom 10.06.2008 hat die Änderung des § 5 (Geschäftsjahr) der Satzung beschlossen.","url":"https://www.handelsregisterbekanntmachungen.de/en/skripte/hrb.php?rb_id=252215\u0026land_abk=by"}

```

**Comment by dchaplinsky** [comment]
That notice has no event_type.
Fixed it.

**Comment by skenaja** [comment]
closed via 7c59ded2ac3eea921339f680080f9c7539d28d5a
